web/components/comments/comments-list.tsx

```diff
@@ -27,24 +28,40 @@ function contractPath(slug: string) {
 
 export function UserCommentsList(props: { user: User }) {
   const { user } = props
+  const pageSize = 50
+  const [pageNum, setPageNum] = useState(0)
+  const numComments = useNumUserComments(user.id)
+  const [pageComments, setPageComments] = useState<
+    {
+      key: {
+        contractId: any
+        contractQuestion: any
+        contractSlug: any
+      }
+      items: unknown[]
```

Comment:
Why are they key fields `any` instead of `string`? Aren't items of type `ContractComment`s?